packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -61,674 +58,643 @@ <h1>Slider</h1>
+            aria-hidden="true"
+          >
+            <span class="iui-slider-tick">0</span>
+            <span class="iui-slider-tick">10</span>
+            <span class="iui-slider-tick">20</span>
+            <span class="iui-slider-tick">30</span>
+            <span class="iui-slider-tick">40</span>
+            <span class="iui-slider-tick">50</span>
+            <span class="iui-slider-tick">60</span>
+            <span class="iui-slider-tick">70</span>
+            <span class="iui-slider-tick">80</span>
+            <span class="iui-slider-tick">90</span>
+            <span class="iui-slider-tick">100</span>
+          </div>
+          <div class="iui-slider iui-range-slider">
+            <div
+              class="iui-slider-track"
+              style="inset-block-end: 20%; inset-block-start: 20%;"
+            ></div>
             <div
               class="iui-slider-thumb"
               role="slider"
-              aria-label="Start Value"
+              tabindex="0"
+              aria-label="Start value"
+              aria-orientation="vertical"
               aria-valuemin="0"
               aria-valuenow="20"
               aria-valuemax="79"
               data-range-max="100"
-            >
-            </div>
-            <output class="iui-tooltip">
-              20
-            </output>
+              style="--iui-slider-thumb-position: 20%;"
+              aria-disabled="true"
+            ></div>
             <div
               class="iui-slider-thumb"
               role="slider"
-              aria-label="End Value"
+              tabindex="0"
+              aria-label="End value"
+              aria-orientation="vertical"
               aria-valuemin="21"
               aria-valuenow="80"
               aria-valuemax="100"
               data-range-min="0"
-            >
-            </div>
-            <output class="iui-tooltip">
-              80
-            </output>
-
-            <div class="iui-slider-ticks">
-              <!-- You could generate the ticks based on your min, max & step values. -->
-              <span class="iui-slider-tick">0</span>
-              <span class="iui-slider-tick">10</span>
-              <span class="iui-slider-tick">20</span>
-              <span class="iui-slider-tick">30</span>
-              <span class="iui-slider-tick">40</span>
-              <span class="iui-slider-tick">50</span>
-              <span class="iui-slider-tick">60</span>
-              <span class="iui-slider-tick">70</span>
-              <span class="iui-slider-tick">80</span>
-              <span class="iui-slider-tick">90</span>
-              <span class="iui-slider-tick">100</span>
-            </div>
+              style="--iui-slider-thumb-position: 80%;"
+              aria-disabled="true"
+            ></div>
           </div>
-          <span class="iui-slider-max">100</span>
         </div>
-
       </div>
     </section>
 
-    <script>
-
+    <!-- <script>
```

Comment:
@rohan-kadkol, could you update this script once more?  Currently it sets `right` or `left` when a handle is dragged.  For the slider, can it instead set [the variables mentioned here](https://github.com/iTwin/iTwinUI/pull/1469#discussion_r1332161230)?  I think the slider & range will have different implementations.  @mayank99, is it even worth it having the sliders work in the HTML/CSS demo page?  Maybe we just remove all of this JS?

Reply:
lets update it to use the CSS variables for now, since it's fairly simple.

in the future, i have ideas for bigger changes

Reply:
> @rohan-kadkol, could you update this script once more?

https://github.com/iTwin/iTwinUI/pull/1469/commits/dd9e702d01973a32385dc46c3fa00594c085f50a Sounds good, I made the change in the script to use `--iui-slider-thumb-position` and `--iui-slider-track-position`.